fmc/src/hand_off.rs

```diff
@@ -230,8 +297,9 @@ impl HandOff {
         self.fht.rt_cdi_kv_hdl = Self::rt_cdi_store(&out);
         self.fht.rt_priv_key_kv_hdl = Self::rt_priv_key_store(&out);
         self.fht.rt_dice_pub_key = out.subj_key_pair.pub_key;
-        self.fht.fmc_cdi_kv_hdl = FHT_INVALID_HANDLE;
-        self.fht.fmc_priv_key_kv_hdl = FHT_INVALID_HANDLE;
         Ok(())
     }
+    pub fn is_valid(&self) -> bool {
```

Comment:
Could you add a comment explaining in what way this is checking validity of the FHT? IIUC, it's just making sure the RT CDI and private key are present in KeyVault. Is that correct?

Reply:
Correct. Upon exit, before FMC hands execution off to RT, it needs to make sure the dice output has been produced.